### PR TITLE
Media Copy Update

### LINKED ISSUE #1757: Update the wording in Reuse content (How to use) section
## Description

We need to replace "Reuse content" title with "How to use" and add a descriptive paragraph to give more context about how to give attribution.

### Desktop (image)

Screen Shot 2022-08-30 at 9 10 55 AM

### Mobile (image)
Screen Shot 2022-08-30 at 9 11 10 AM

### The image description text:
Visit the image’s website to download and use it. Make sure to credit the creator by showing the attribution information where you are sharing your work.

### The audio description text:
Visit the audio’s website to download and use it. Make sure to credit the creator by showing the attribution information where you are sharing your work.

## Mockups
Presentation: [page 7](https://www.figma.com/proto/ClEKESZzZxudMoHbHq3A66/Copy-improvements-in-the-search-flow?page-id=26%3A13657&node-id=435%3A4760&viewport=461%2C537%2C0.3&scaling=min-zoom&starting-point-node-id=427%3A9251)
Figma: [desktop](https://www.figma.com/file/ClEKESZzZxudMoHbHq3A66/Copy-improvements-in-the-search-flow?node-id=418%3A5722) and [mobile](https://www.figma.com/file/ClEKESZzZxudMoHbHq3A66/Copy-improvements-in-the-search-flow?node-id=418%3A3288)

@panchovm: Please note that Grammarly proposed to remove the second `to` from this sentence:
Visit the image’s website to download and ~~to~~ use it.

### PR DESCRIPTION
**Pull request by ramadanomar**
## Fixes
Fixes #1757 by @obulat.

## Description
I still have to add the icon to Get this Image/Audio button. Should i add a span with the icon inside or use VIconButton component?

### CODE REVIEW

**Review by obulat — COMMENTED** [review]
The updates to the "Get this <media>" were merged in https://github.com/WordPress/openverse-frontend/pull/1741, so you can remove them here.

**Comment by panchovm** [comment]
Coming to say that the text must be "Visit the image’s website to download and use it" as suggested by @obulat

**Comment by ramadanomar** [comment]
Hey @panchovm, the text should be good now. Thanks for confirming the change !

File: src/components/VMediaInfo/VMediaReuse.vue [hunk 1]
```diff
@@ -3,10 +3,17 @@
     :aria-label="$t('media-details.reuse.title').toString()"
     class="media-reuse"
   >
-    <h2 class="mb-6 text-2xl md:text-3xl">
+    <h2 class="mb-4 text-2xl md:text-3xl">
       {{ $t('media-details.reuse.title') }}
     </h2>
-    <div v-if="media.license_url" class="grid gap-6 md:grid-cols-2">
+    <span class="text-base font-normal leading-[130%]">
```
Comment (zackkrida) [inline src/components/VMediaInfo/VMediaReuse.vue:9]:
```suggestion
    <p class="text-base font-normal leading-[130%]">
```

File: src/components/VMediaInfo/VMediaReuse.vue [hunk 2]
```diff
@@ -3,10 +3,17 @@
...
+      {{
+        $t('media-details.reuse.description', {
+          media: media.frontendMediaType,
+        })
+      }}
+    </span>
```
Comment (zackkrida) [inline src/components/VMediaInfo/VMediaReuse.vue:15]:
```suggestion
    </p>
```

File: src/locales/en.json [hunk 3]
```diff
@@ -397,7 +397,8 @@
   },
   "media-details": {
     "reuse": {
-      "title": "Reuse content",
+      "title": "How to use",
+      "description": "Visit the {media}'s website to download and to use it. Make sure to credit the creator by showing the attribution information where you are sharing your work.",
```
Comment (zackkrida) [inline src/locales/en.json:401]:
```suggestion
      "description": "Visit the {media}'s website to download and use it. Make sure to credit the creator by showing the attribution information where you are sharing your work.",
```

**Review by zackkrida — CHANGES_REQUESTED** [review]
Some small fixes, but looks nice.

**Review by obulat — APPROVED** [review]
LGTM!

**Review by zackkrida — APPROVED** [review]

**Comment by obulat** [comment]
I am going to fix Playwright test failures and merge this PR, @ramadanomar.

**Comment by panchovm** [comment]
I am seeing one style mistake.

On mobile, the descriptive paragraph below section title is font-size `14px` and line-height `18px` instead of `14px` and `18px` respectively, as `Label regular` style was defined.

**Pull request merged**